### ISSUE TITLE
Batch size vs. train steps experiments

### ISSUE THREAD

**Issue by LuggiStruggi** [hufflepuff79/online_stagnation_teaching#4]
As hinted in the paper, increasing the batch size can reduce the computation time by a lot.
Choose different batch sizes while changing the amount of steps accordingly  (double batch size -> half steps) to have the same data diversity. This may also require to change the learning rate of ADAM.
The resources are 125GB cluster + 52GB cloud. This is enough for 4 simultaneous agents, each with 5 checkpoints loaded in the buffer simultaneously.
1. Figure out how many of these should be used for this experiment.
2. Write parameter files for each experiment
3. Run the experiments.
